packages/sdk/src/types/Campaigns.ts

```diff
@@ -124,37 +502,61 @@ export type CampaignsAddCertainVoucherParams = Pick<
 	'category' | 'redemption' | 'metadata' | 'additional_info'
 >
 
-export type CampaignsAddVoucherResponse = Pick<
-	VouchersResponse,
-	| 'code'
-	| 'object'
-	| 'campaign'
-	| 'category'
-	| 'created_at'
-	| 'type'
-	| 'discount'
-	| 'gift'
-	| 'start_date'
-	| 'expiration_date'
-	| 'publish'
-	| 'redemption'
-	| 'active'
-	| 'additional_info'
-	| 'metadata'
-	| 'assets'
->
+export type CampaignsAddVoucherResponse = VoucherObject | AsyncActionCreateResponse
 
 export type CampaignsAddCertainVoucherResponse = CampaignsAddVoucherResponse
 
-export type CampaignsImportVouchers = Pick<
```

Comment:
It looks like you have changed the `CampaignsImportVouchers` type. This type is exported, so we need to treat them as a public API interface -  any non-backwards compatible changes need to be described in detail. It can not be marked as a minor update.

Reply:
We need to go through all updated types and write down the following:
- if we just extended the type in a backwards-compatible way by adding missing props
- if we have changed the type like in the example above, we need to describe "why", how to adopt code using the old type to the new one and describe all differences in details